src/main/java/frc/robot/util/MoUnits.java

```diff
@@ -29,6 +29,6 @@ public static class EncoderAngle extends Unit<EncoderAngle> {
     public static Per<EncoderAngle, Distance> EncoderTicksPerCentimeter = EncoderTicks.per(Units.Centimeters);
     public static Per<EncoderAngle, Angle> EncoderTicksPerRotation = EncoderTicks.per(Units.Rotations);
     public static Per<EncoderAngle, Angle> EncoderTicksPerRadian = EncoderTicks.per(Units.Radians);
-
+    public static Velocity<Angle> RotationsPerMinute = Units.Rotations.per(Units.Minutes);
```

Comment:
This unit shouldn't be necessary. We keep all our own calculations in rotations per second.

While we do need to format into rotations per minute to pass to the Spark Maxes, that's handled by the MoEncoder's ability to determine the motor controller's "velocity base unit".